src/libraries/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSourceActivity.cs

```diff
@@ -317,6 +317,18 @@ public sealed class Counter<T> : Instrument<T> where T : struct
         internal Counter(Meter meter, string name, string? unit, string? description) :
                         base(meter, name, unit, description) {  throw null; }
     }
+    public sealed class UpDownCounter<T> : Instrument<T> where T : struct
+    {
+        public void Add(T delta) {  throw null; }
```

Comment:
curious, is there a reason to have two spaces after `{` while one space before `}`?

Reply:
No reason. I'll remove the extra space.